pylot/component_creator.py

```diff
@@ -70,10 +70,10 @@ def add_obstacle_detection(center_camera_stream,
         if any('efficientdet' in model
                for model in FLAGS.obstacle_detection_model_names):
             logger.debug('Using EfficientDet obstacle detector...')
-            obstacles_streams = pylot.operator_creator.\
+            obstacles_stream = pylot.operator_creator.\
```

Comment:
Could you add type annotations to `def add_obstacle_detection(center_camera_stream: erdos.Stream, ...) -> Tuple[erdos.Stream, erdos.Stream]`?